scripts/install_xcpng.py

```diff
@@ -230,11 +244,17 @@ def main():
         host2 = pool2.master
         host2.inventory = host2._get_xensource_inventory()
         check_mac_address(host2, mac_address)
-        logging.info('New host is started and enabled')
-        if host2.xcp_version != version.parse(xcp_version):
+        logging.info(f'New host is started and enabled in version:{host2.xcp_version}')
+        if args.action == 'restore' and host2.xcp_version < version.parse(xcp_version):
+            logging.info(f'The host {vm_ip_address} is in version : {host2.xcp_version}')
```

Comment:
This doesn't seem useful to me. This repeats what the previous line says already.